### ISSUE TITLE
Remote Environments/Service Brokers are not sensitive to Authorizations

### ISSUE THREAD

**Issue by akrausesap** [kyma-project/kyma#258]
Confirm these statements before you submit the issue:

- [X] I have searched open and closed issues for duplicates.
- [X] I have read the contributing guidelines.
---

**Description**

Dear Colleagues,

after a new cluster was setup, I was setup only as user. Hence I was getting a 403 error everywhere in the console which was good. However when browsing the Administration menu I could still see Service Brokers and Remote Environments (including a working 1 click connect). 

Can you check this behavior?

Cheers

Andreas

**Expected result**

User receives 403

**Actual result**

User was able to browse Service Brokers and Remote Environments

**Steps to reproduce**

Remove Administrative Permission from user and go to Administration in Cluster

**Troubleshooting**

N/A

**Comment by magicmatatjahu** [comment]
The authentication by Istio is not implemented yet.

**Comment by pkosiec** [comment]
As confirmed with @derberg, all we have to do is:

- [ ] Secure IDP Presets tab in administration view
- [ ] Secure Remote Environments tab in administration view

Securing the views is understood by securing queries for RE and IDP presets

**Comment by pkosiec** [comment]
Blocked by https://github.com/kyma-project/kyma/issues/1789

**Comment by derberg** [comment]
still valid, and because the authorization is not fully supported in backed we can implement the fix

**Comment by pkosiec** [comment]
I'm closing this as It will be resolved in #3062. Please follow the new issue to see the progress.
